### DISCUSSION TITLE
Rotation Problems

### DISCUSSION THREAD

**Discussion by Eddie-Edward** [mjansen4857/pathplanner#574] [Q&A]
Could someone help me to understand why when I write a path that has rotation points, the model that runs doesn't actually rotate to the rotation points made? 
![image](https://github.com/mjansen4857/pathplanner/assets/111027605/915528bc-f5af-40f0-bec8-7b9eb53a63b3)

**Comment by rakar** [comment]
I'm assuming you are set for Holonomic. On thing that could be an issue is if your max module speed is too low. The system favors translation over rotation, so if the module speed is low relative to your path speed rotation gets lost. You may want to provide more information (AutoBuilder config, path constraints, etc.).
